osu.Game/Updater/UpdateManager.cs

```diff
@@ -16,6 +17,8 @@ namespace osu.Game.Updater
     /// </summary>
     public class UpdateManager : CompositeDrawable
     {
+        public virtual bool CanPerformUpdate => false;
```

Comment:
no xmldoc. what does this even mean?

Reply:
why is this not including the IsDeployedBuild check?

Reply:
After I committed it, I myself doubted the name of this property, a more appropriate name would be "CanCheckForUpdate".  That name would make more sense and is self-explanatory.

Reply:
but why is IsDeployedBuild not included and how does that check differ from the one you added?